comps/llms/text-generation/bedrock/llm.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import os
+from typing import Union
+
+import boto3
+from fastapi.responses import StreamingResponse
+from langchain_aws import BedrockLLM, ChatBedrock
+
+from comps import (
+    CustomLogger,
+    GeneratedDoc,
+    LLMParamsDoc,
+    SearchedDoc,
+    ServiceType,
+    opea_microservices,
+    register_microservice,
+    register_statistics,
+    statistics_dict,
+)
+from comps.cores.proto.api_protocol import ChatCompletionRequest
+
+bedrock_runtime = boto3.client(service_name="bedrock-runtime", region_name="us-east-1")
```

Comment:
Parameterize region too please
```suggestion
region = os.getenv("AWS_DEFAULT_REGION", "us-east-1")
bedrock_runtime = boto3.client(service_name="bedrock-runtime", region_name=region)
```